tictactoe/src/components/Game.js

```diff
@@ -1,16 +1,47 @@
 import React, { useState } from "react";
 import Board from "./Board";
 import "./Game.css";
-import { calculateWinner } from "../helpers/calculateWinner";
 
 export default function Game() {
   const [playerMovePositions, setPlayerMovePosition] = useState(
     Array(9).fill("")
   );
+  const [currentPlayer, setCurrentPlayer] = useState("X");
+  const [isOver, setGameOver] = useState(false);
 
-  return (
-    <div>
-      <Board playerMovePositions={playerMovePositions} />
-    </div>
-  );
+  function restartGame() {
+    setPlayerMovePosition(Array(9).fill(""));
+    window.location.reload();
+  }
+
+  if (isOver) {
+    return (
+      <div>
+        <Board
+          playerMovePositions={playerMovePositions}
+          currentPlayer={currentPlayer}
+          setCurrentPlayer={setCurrentPlayer}
+          setPlayerMovePosition={setPlayerMovePosition}
+          isOver={isOver}
+          setGameOver={setGameOver}
+        />
+
+        <button onClick={() => restartGame()}>Restart Game</button>
+      </div>
+    );
+  } else {
+    return (
+      <div>
+        <Board
+          playerMovePositions={playerMovePositions}
+          currentPlayer={currentPlayer}
+          setCurrentPlayer={setCurrentPlayer}
+          setPlayerMovePosition={setPlayerMovePosition}
+          setGameOver={setGameOver}
+        />
+
+        <button onClick={() => restartGame()}>Restart Game</button>
+      </div>
+    );
+  }
```

Comment:
For conditional rendering

```
if(isOver) {
    return (<JSX/>)
}

if(anotherConditionIfYouHaveOne) {
    return (<JSX/>
}

// the base return
return (<JSX/>)
```

the else statement clogs things up and is generally uncessary cause the return will handle things.

Reply:
Did notice it on one of the features I was working on. I will refactor the code and reduce them.